es/node.js

```diff
@@ -90,14 +90,14 @@ const Node = stampit({
 }, Swagger, {
   async init ({ forceCompatibility = false }) {
     const { nodeRevision: revision, genesisKeyBlockHash: genesisHash, networkId } = await this.api.getStatus()
-    if (!semver.satisfies(this.version, COMPATIBILITY_RANGE) && !forceCompatibility) throw new Error(`Unsupported node version ${this.version}. Supported: ${COMPATIBILITY_RANGE}`)
+    if (!semver.satisfies(this.version.split('-')[0], COMPATIBILITY_RANGE) && !forceCompatibility) throw new Error(`Unsupported node version ${this.version}. Supported: ${COMPATIBILITY_RANGE}`)
```

Comment:
will this still work with a normal semver number, like 2.0.0 ?

Reply:
yes